test/integrations/routes/users.test.js

```diff
@@ -2,46 +2,51 @@
 import chai, { expect } from 'chai';
 import chaiHttp from 'chai-http';
 import Bluebird from 'bluebird';
+import faker from 'faker';
+import sinon from 'sinon';
 import app from '../../../index';
 import models from '../../../models';
 import {
   DUMMY_USER,
   STATUS, MESSAGE,
   FIELD
 } from '../../../helpers/constants';
+import Mail from '../../../helpers/sendMail';
 
 chai.use(chaiHttp);
 let dummyUser;
 
 describe('API endpoint: /api/users', () => {
-  before(() => {
-    // synchronize and create tables
-    models.sequelize.sync();
-  });
   beforeEach(() => {
     // reset the dummy user data
     dummyUser = { ...DUMMY_USER };
-    // drop tables
+    // clear database table
     Bluebird.all([
```

Comment:
this part should be after the test has finished. You can truncate. But I strongly advise against this. The profile table and article table depends on this. Hence it is not cool if the other tests run and cannot find the data in the user table anymore because you truncated it. I strongly recommend we remove the truncate from the users test file.

Reply:
I understand this may be a concern considering that the profile data depends on the user. Moving forward I think its a good idea to retain the data. Changes have been made.

Reply:
Thank you.